apps/web/src/components/TrendingItemSuggestion/TrendingItemSuggestion.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import './TrendingItemSuggestion.scss';
+import { ReactComponent as GoCircleIcon } from '../../assets/goCircleIcon.svg';
+import Icon from '../Icon/Icon';
+
+interface Props {
+  metaData: {
+    URL: string;
+    trendingSuggestionHeading: string;
+    trendingSuggestionSummary: string;
+    targetSubreddit: {
+      subRedditIcon: React.ReactElement;
+      subRedditName: string;
+    };
+  };
+  handleClick: () => void;
+  customTestId?: string;
+}
+
+const TrendingItemSuggestion = ({
+  metaData,
+  handleClick,
+  customTestId = 'list-item',
+}: Props) => {
+  const {
+    URL,
+    trendingSuggestionHeading,
+    trendingSuggestionSummary,
+    targetSubreddit,
+  } = metaData;
+  const { subRedditIcon, subRedditName } = targetSubreddit;
+
+  return (
+    <li
+      onClick={handleClick}
+      className={!metaData ? 'loading' : ''}
```

Comment:
I'm not sure this will work the way that you think it will. `metaData` is a required prop, so this will always be truthy. We are going to be using `ReactQuery` for out FE data fetching / caching and that will expose a the async lifecycle values that can be used to handle an `isLoading` prop. This prop can be consumed by any component that is dependent on async data. 

We will do the actual data fetching in `TrendingItemSuggestion`'s parent component - `TrendingSuggestionList`. It will look something like this:

```TrendingSuggestionList.tsx
import { useQuery } from 'react-query'
import { fetchTrendingSuggestions } from './TrendingSuggestions.api.ts'

// fetchTrendingSuggestions is an async function (probably Axios) that makes the actual request to the endpoint
// useQuery is a custom hook exposed by `react-query`
// data is the same as `typeof fetchTrendingSuggestions`, in this case an array of `TrendingSuggestionItems`, // which should have the same shape as the `metaData` object
const { isFetching, isFetched, data } =  useQuery('trendingSuggestions', fetchTrendingSuggestions); 

return(
{data.map((trendingSuggestionData: TrendingSuggestionData)) => (
  <TrendingItemSuggestion 
    isLoading={!isFetched || isFetching}
    ...trendingSuggestionData
  />
)}
)
```

`react-query` provides us with a wrapper for fetching our data as well as an API for dealing with the async lifecycle all in a very declarative way.

Reply:
@willcary Want to make sure you see this comment.